Automattic/jetpack
# Search: Fix responsive layouts.

Fixes #26814 

#### Changes proposed in this Pull Request:


- Fix line height of H2 so the "Your usage" line wraps cleanly.
- Fix usage meters to wrap as needed on smaller screens.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Visit the updated Search dashboard.
2. Open the Dev Tools and test with different screen sizes.
3. Should properly wrap on smaller screen sizes.

Before:
![MiniBusted](https://user-images.githubusercontent.com/40267301/195597538-d7fd946f-54ca-4872-924f-6e4096ded352.png)

After:
![MiniFixed](https://user-images.githubusercontent.com/40267301/195597572-bf3b76c5-c61b-4b9d-a0c0-79b1e13f8bf1.png)
